torchao/utils.py

```diff
@@ -386,7 +386,9 @@ class MyTensor(torch.Tensor):
        func in cls._ATEN_OP_OR_TORCH_FN_TABLE:
         return cls._ATEN_OP_OR_TORCH_FN_TABLE[func](func, types, args, kwargs)
 
-    raise NotImplementedError(f"{cls.__name__} dispatch: attempting to run unimplemented operator/function: {func}")
+    arg_types = tuple(type(arg) for arg in args)
+    kwarg_types = {k: type(arg) for k, arg in kwargs}
```

Comment:
Just came across this today. This line is missing `.items()` i.e. should be `kwargs.items()`